package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "scripts": {
+    "sass": "node compile-scss.js"
+  },
+  "devDependencies": {
+    "sass": "^1.52.3"
+  },
+  "dependencies": {
+    "govuk-frontend": "^4.3.1"
```

Comment:
I wonder if some instructions on what to do when there is an update to `govuk-frontend` would be useful?